src/components/buttons/ha-call-service-button.js

```diff
@@ -69,10 +69,16 @@ class HaCallServiceButton extends EventsMixin(PolymerElement) {
           el.$.progress.actionSuccess();
           eventData.success = true;
         },
-        function () {
+        function (err) {
+          if (err?.error?.message && err.error.message === "Connection lost") {
```

Comment:
We should only do that for specific services, not for all?

Reply:
In which case would other services cause this?

Reply:
If the connection is lost? 😄

Reply:
vaccum.clean?